src/app/executions/tasks-runtime/tasks-runtime.component.scss

```diff
@@ -13,7 +13,7 @@
     background: #dde4e6;
 
     .highlight {
-      border-radius: 2px;
+      //border-radius: 1px;
```

Comment:
1) Remove unused style
2) We can have a combined rule for 'highlight' now that we separated to 2 classes:
```scss
.highlight {      
      transition: filter .3s ease-in-out;
      background-color: $entity-completed;

      &:hover {
        filter: brightness(1.1);
      }

      &.waiting {
        background-color: $entity-waiting;
      }
    }
```
so you could delete the `.waiting-highlight` rule as well.

Reply:
done